Sources/macro.js

```diff
@@ -1150,6 +1152,10 @@ export function proxy(publicAPI, model) {
   publicAPI.getPropertyByName = (name) =>
     getProperties().find((p) => p.name === name);
 
+  publicAPI.getPropertyDomainByName = (name) => {
+    return propertyMap[name].domain;
+  };
+
```

Comment:
need to be reformated

`publicAPI.getPropertyDomainByName = (name) => propertyMap[name].domain;`